ExecCpp/Make.PeleC

```diff
@@ -108,16 +108,24 @@ ifeq ($(USE_EB), TRUE)
 endif
 
 ifeq ($(USE_PARTICLES), TRUE)
-  DEFINES+=-DAMREX_PARTICLES
   Pdirs += Particle
   DEFINES+=-DSPRAY_FUEL_NUM=$(SPRAY_FUEL_NUM)
   ifeq ($(USE_SPRAY_SOA), TRUE)
     DEFINES+=-DUSE_SPRAY_SOA
   endif
   Bpack   += $(PELEC_MP_HOME)/Source/PP_Spray/Make.package
   Blocs   += $(PELEC_MP_HOME)/Source/PP_Spray
-  INCLUDE_LOCATIONS += $(AMREX_HOME)/Src/Particle
-  VPATH_LOCATIONS   += $(AMREX_HOME)/Src/Particle
+endif
+
+ifeq ($(USE_SOOT), TRUE)
+# Ensure the number of soot moments is correct
+    ifeq (,$(filter $(NUM_SOOT_MOMENTS),3 6))
+        $(error NUM_SOOT_MOMENTS must be 3 or 6)
+    endif
+  DEFINES+=-DSOOT_MODEL
+  DEFINES+=-DNUM_SOOT_MOMENTS=$(NUM_SOOT_MOMENTS)
+  Bpack   += $(PELEC_MP_HOME)/Source/Soot_Models/Make.package
```

Comment:
Is PeleC_MP a submodule of PeleC now? Maybe it should be?

Reply:
It is a submodule.